bin/fzp/command-validate.go

```diff
@@ -145,13 +145,16 @@ func validateFile(c *cli.Context, src string) error {
 		// fmt.Println("call validateFile", src)
 		fzpData, _, err := fzp.ReadFzp(src)
 		if err != nil {
-			fmt.Printf("validator failed @ %v\n", err)
+			fmt.Printf("validator failed @ %v, %s\n ", err, src)
```

Comment:
`err` does not need `%v`. You can use directly a string `%s`

Reply:
@ovidiub13 okay, then let us change the `%v` to `%s` at this PR also.

Reply:
```suggestion
			fmt.Printf("validator failed @ %s, %s\n ", err, src)
```